### DISCUSSION TITLE
Why can't I convert BeeWare briefcase to an Apk file?

### DISCUSSION THREAD

**Discussion by Slonit** [beeware/briefcase#1654] [Q&A] [answered]
I'm working using the BeeWare Tutorial website and got to the first lesson, but decided not to do the second. I moved on to the third lesson and everything seemed to be going fine, I inserted commands into the terminal, but on the last one it just doesn’t work. Below are the photos: the first one is as it should be, the second one is like mine. As a result, the android folder should appear, but it doesn’t appear for me at all, I even replaced the commands like “briefcase package” with “briefcase build android”. In any case, the same thing. Help me please!
![Безымянны3й](https://github.com/beeware/briefcase/assets/143531646/02b32c07-3491-4151-8729-3a4d7bae1880)
![Безымянный2](https://github.com/beeware/briefcase/assets/143531646/b3961a81-6717-4661-be8c-4c06110f5ec7)

**Comment by rmartin16** [comment, marked as answer]
I would first verify that you can, in fact, create a file or folder in the file path mentioned for the `PermissionError`. In case something went wrong somehow, you can just delete the `Appdata\Local\BeeWare` directory altogether and start over with Briefcase.

However, I doubt this is actually a "permission issue" since writing to `AppData` is a pretty normal thing. Given the name of the file, it seems potentially more likely an anti-virus system prevented filesystem access since it might have perceived Briefcase to be trying to smuggle in an executable file. So, I would review any installed anti-virus to see if it did block this and take steps for it to allow Briefcase to run without any interruptions.

**Reply by Slonit** [reply]
Handsome, thank you very much, thank you, everything worked out!
